docs/advanced-topics/stack-monitoring.asciidoc

```diff
@@ -72,6 +72,8 @@ You can also enable Stack Monitoring on Elasticsearch only or on Kibana only. In
 
 == When to use it
 
+CAUTION: Minimum supported Stack version is 7.14.0. 
```

Comment:
I think we prefer use a `NOTE` for this kind of information (example: https://www.elastic.co/guide/en/cloud-on-k8s/master/k8s-install-helm.html#k8s-install-helm) but I would like to have @alaudazzi's opinion.
```suggestion
NOTE: The minimum supported Stack version is 7.14.0. 
```

Reply:
Thanks! Indeed NOTE is much better! ❤️

Reply:
I committed the suggestion for the time being but please let me know if we need to update more.